### DISCUSSION TITLE
sumatra creating shortcuts

### DISCUSSION THREAD

**Discussion by alk444** [sumatrapdfreader/sumatrapdf#4470] [Q&A] [answered]
how do i stop sumatra from creating shortcuts, to the documents i open, on my desktop?
win10pro64 22h2; sumatra v3.5.2

**Comment by kjk** [comment, marked as answer]
Sumatra doesn't create such shortcuts so you would have to provide more information, including a screenshot of what you're talking about.

**Reply by alk444** [reply]
now that i look closer, i see they are not shortcuts, but are copies of the downloads. apparently firefox is doing it.
i withdraw the ?
